joomla-projects/privacy-framework
# Log user registration action

Pull Request for Issue #170 .

### Summary of Changes
This PR adding support for logging user registration action on a Joomla website


### Testing Instructions
1. Apply patch
2. Try to register for an account on frontend of the site
3. Check User Actions Log, make sure the action is logged (Something like User test registered for an account)
